### PR TITLE
docs: release update

### PR DESCRIPTION
**Pull request by maciejmajek**
## Purpose

The first major RAI version is around the corner :).

## Proposed Changes

Documentation update

## Issues

- Links to relevant issues

## Testing

- How was it tested, what were the results?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 # RAI
 
 > [!IMPORTANT]  
-> **RAI is in beta phase now, expect friction. Early contributors are the most welcome!** \
-> **RAI is developing fast towards a glorious release in time for ROSCon 2024.**
+> **RAI is in beta phase now, expect friction. Early contributors are the most welcome!**
```
Comment (maciejmajek) [inline README.md]: to be removed

File: README.md
Comment (maciejmajek) [inline README.md:1]: `## Running RAI`
sections needs a refactor

File: src/rai/rai/apps/task_executor.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def get_latest_data():
     it = 0
     logger.info("Entering the task execution loop.")
     while it < max_iters:
-        logger.info(f"Task execution loop iteration: {it+1}/{max_iters}")
+        logger.info(f"Task execution loop iteration: {it + 1}/{max_iters}")
```
Comment (maciejmajek) [inline src/rai/rai/apps/task_executor.py:127]: Misc change that would trigger pre-commit on u24

**Comment by maciejmajek** [comment]
@boczekbartek could you please update the `rai/docs/demos/rosbot_xl.md` and push the changes here?

File: README.md [hunk 3]
```diff
@@ -38,38 +37,43 @@ The RAI framework aims to:
...
+- Limitations of LLMs and VLMs in use apply: poor spatial reasoning, hallucinations, jailbreaks, latencies, costs, ...
+- Resource use (memory, CPU) is not addressed yet.​
+- Requires connectivity and / or an edge platform.​
 
 ## Table of Contents
 
 - [Features](#features)
 - [Setup](#setup)
-- [Usage examples (demos)](#planned-demos)
+- [Usage examples (demos)](#roscon-2024-demos)
```
Comment (boczekbartek) [inline README.md]:
```suggestion
- [Usage examples (demos)](#simulation-demos)
```

File: README.md [hunk 4]
```diff
@@ -162,15 +142,15 @@ Once you know your way around RAI, try the following challenges, with the aid th
 - Implement additional tools and use them in your interaction.
 - Try a complex, multi-step task for your robot, such as going to several points to perform observations!
 
-Soon you will have an opportunity to work with new RAI demos across several domains.
-
-### Planned demos
+### ROSCon 2024 demos
```
Comment (boczekbartek) [inline README.md]:
```suggestion
### Simulation demos
```

File: docs/human_robot_interface/text_interface.md [hunk 5]
```diff
@@ -1,12 +1,20 @@
...
-> Agent's responses can take longer time for complex tasks.
+> The agent's responses may take longer for complex tasks.
+
+## Customization
+
+Currently, customization capabilities are limited due to the internal API design. We are planning to deliver a solution for seamless expansion in the near future.
+
+If you want to customize the available tools, you can do so by editing the `src/rai_hmi/rai_hmi/agent.py` file.
+
+If you have a RaiStateBasedLlmNode running (see e.g., examples/rosbot-xl-generic-node-demo.py), the Streamlit GUI will communicate with the running node via task_tools defined in the `rai_hmi/rai_hmi/agent.py` file.
```
Comment (boczekbartek) [inline docs/human_robot_interface/text_interface.md]:
```suggestion
If you have a RaiStateBasedLlmNode running (see e.g., [examples/rosbot-xl-demo.py](examples/rosbot-xl-demo.py)), the Streamlit GUI will communicate with the running node via task_tools defined in the `rai_hmi/rai_hmi/agent.py` file.
```

**Review by boczekbartek — APPROVED** [review]

**Review by boczekbartek — APPROVED** [review]

**Pull request merged**